pkg/labels/selector_test.go

```diff
@@ -45,6 +45,8 @@ func TestSelectorParse(t *testing.T) {
 		"!x",
 		"x>1",
 		"x>1,z<5",
+		"indent.com/some/label=indent.com/some/value",
+		"indent.com/some/label=some@email.com",
```

Comment:
would adding a set-based test help?
```suggestion
		"indent.com/some/label=some@email.com",
		"indent.com/resource/email in (abc+def@email.com, some@email.com)",
```